protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -275,6 +275,7 @@ func buildEtcdBackupManagerContainer(c *EtcdCluster) *v1.Container {
 		command = append(command, "--client-ca-file", c.TLSCA)
 		command = append(command, "--client-cert-file", c.TLSCert)
 		command = append(command, "--client-key-file", c.TLSKey)
+		command = append(command, "--listen-metric-urls", "https://__address__:"+c.MetricsPort)
```

Comment:
You'll need to convert `c.MetricsPort` to a string here as currently govet is failing as it's trying to do maths on the `"https://__address__:"` string.

Reply:
Good spot, thanks. Solved.